crates/rolldown/src/module_loader/runtime_module_task.rs

```diff
@@ -20,6 +20,28 @@ use crate::{
   utils::tweak_ast_for_scanning::PreProcessor,
 };
 
+const RUNTIME_BASE_JS: &str = include_str!("../runtime/runtime-base.js");
+const RUNTIME_TAIL_JS: &str = include_str!("../runtime/runtime-tail.js");
+const RUNTIME_HEAD_NODE_JS: &str = include_str!("../runtime/runtime-head-node.js");
+const RUNTIME_TAIL_NODE_JS: &str = include_str!("../runtime/runtime-tail-node.js");
+
+fn get_runtime_js() -> String {
+  let mut ret = String::with_capacity(RUNTIME_BASE_JS.len() + RUNTIME_TAIL_JS.len());
+  ret.push_str(RUNTIME_BASE_JS);
+  ret.push_str(RUNTIME_TAIL_JS);
+  ret
+}
+
+fn get_runtime_js_with_node_platform() -> String {
```

Comment:
It's fine to concatenate them at runtime, because the runtime module only generated once.